src/components/Picker/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import * as style from './Picker.scss';
+export interface Props {
+  image?: string;
+  nameBefore?: string;
+  bold?: string;
+  nameAfter?: string;
+  email?: string;
+  isHighlighted?: boolean;
+}
+
+class Card extends React.Component<Props, {}> {
+  constructor(props: any) {
+    super(props);
+  }
+  render() {
+    let cardBackground;
```

Comment:
cardBackground could be a const and have the if else statement a ternary. That way you're never re-assigning anything.